rx/__init__.py

```diff
@@ -110,35 +118,43 @@ def catch_with_iterable(sources: Iterable[Observable]) -> Observable:
         >>> res = rx.catch(src for src in [xs, ys, zs])
 
     Args:
-        sources: an Iterable of observables. Thus a generator is accepted.
+        sources: An Iterable of observables; thus, a generator can also
+            be used here.
 
     Returns:
-        An observable sequence containing elements from consecutive
-        source sequences until a source sequence terminates
-        successfully.
+        An observable sequence containing elements from consecutive observables
+        from the sequence of sources until one of them terminates successfully.
     """
+
     from .core.observable.catch import _catch_with_iterable
     return _catch_with_iterable(sources)
 
 
-def create(subscribe: Callable[[typing.Observer, Optional[typing.Scheduler]], typing.Disposable]):
-    """Create observable from subscribe function.
+def create(subscribe: typing.Subscription) -> Observable:
+    """Creates an observable sequence object from the specified
+        subscription function.
 
     .. marble::
         :alt: create
 
         [     create(a)    ]
         ---1---2---3---4---|
 
+    Args:
+        subscribe: [Optional] Subscription function.
```

Comment:
Should `subscribe` be qualified as `Optional` with `None` as a default value in the signature? Don't know if it's worth to call `create` with no subscribe function thougth.

Reply:
Ah, good catch. Sloppy/paste, I guess... I'll amend.